airbyte-cdk/python/airbyte_cdk/sources/declarative/decoders/decoder.py

```diff
@@ -15,11 +15,17 @@ class Decoder:
     Decoder strategy to transform a requests.Response into a Mapping[str, Any]
     """
 
+    @property
```

Comment:
nit: let's leave this as an abstract method and force the developers to think about whether the response should be streamed or not when implementing a decoder

Reply:
done